src/vmm/src/devices/virtio/generated/virtio_blk.rs

```diff
@@ -52,34 +37,8 @@ pub const VIRTIO_BLK_T_FLUSH: u32 = 4;
 pub const VIRTIO_BLK_T_GET_ID: u32 = 8;
 pub const VIRTIO_BLK_T_DISCARD: u32 = 11;
 pub const VIRTIO_BLK_T_WRITE_ZEROES: u32 = 13;
-pub const VIRTIO_BLK_T_SECURE_ERASE: u32 = 14;
```

Comment:
Why we lose these constants with the change?

Reply:
I think its probably because I ran ./tools/bindgen.sh on my laptop, which must have a different version of the host kernel compared to what they were generated from the last time

Reply:
Do you think it's worth holding onto these incase we need them in the future? Save us having to bindgen again :D

Reply:
I don't think we'll need them, but let me readd them anyway, nicer to not needlessly touch these indeed

Reply:
done